index.html

```diff
@@ -11,11 +11,20 @@
         <span><img class="icon" src="./assets/menu.svg" height="50" width="50"></i></span>
         <span><h1 class="header-text">IdeaBox</h1></span>
       </nav>
-      <div class="dropdown hidden">
-        <h1 class="dropdown-ideabox">IdeaBox</h1>
-        <i class=""></i>
-        <p>Filter Starred Ideas</p>
-        <button class="starred-ideas-button">Show Starred Ideas</button>
+      <div class="dropdown-page hidden">
+        <div class="dropdown hidden">
+          <div class="header-alignment">
+            <div class="dropdown-header">
+              <span><img class="dropdown-icon" src="./assets/menu-close.svg" height="40" width="40"></i><span>
+            </div>
+            <div>
+              <h1 class="dropdown-ideabox">IdeaBox</h1>
+            </div>
+          </div>
+          <p class="filter-idea-text">Filter Starred Ideas</p>
+          <button class="starred-ideas-button">Show Starred Ideas</button>
+        </div>
+        <div class="opaque">stuff</div>
```

Comment:
Should "stuff" still be there? We can remove that in the next push.

Reply:
Great catch! I've removed it from my code.